packages/cli/commands/init.js

```diff
@@ -107,8 +109,11 @@ exports.handler = async options => {
     auth: authType = PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
     c,
     account: optionalAccount,
+    useRootConfig,
   } = options;
-  const configPath = (c && path.join(getCwd(), c)) || getConfigPath();
+  const configPath =
```

Comment:
Might make sense to error if the `-c` flag and `--useRootConfig` are both passed in

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1123/commits/7f9053a5b2f5659aad7c4203bd05278e774ed862